backends/gstreamer/src/player.rs

```diff
@@ -324,32 +364,97 @@ impl GStreamerPlayer {
             observers: Arc::new(Mutex::new(PlayerEventObserverList::new())),
             renderers: Arc::new(Mutex::new(FrameRendererList::new())),
             is_ready: Arc::new(Once::new()),
+            gl_context: RefCell::new(None),
+            gl_display: RefCell::new(None),
         }
     }
 
+    fn set_bus_sync_handler(&self, pipeline: &gst::Element) -> Result<(), PlayerError> {
+        use gst::ElementExt;
```

Comment:
Pattern is usually to `use gst::prelude::*;` at the top and never worry about the extension traits